### PR TITLE
Add dependabot

### PR DESCRIPTION
**Pull request by spaceo**
#### Link to issue

https://reload.atlassian.net/browse/DDFMINI-17

#### Description

Adds dependabot for tracking dependency changes.

#### Checklist

- [x] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [x] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [x] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

### CODE REVIEW

**Review by Adamik10 — APPROVED** [review]
👍

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+version: 2
```
Comment (kasperg) [inline .github/dependabot.yml:1]: Consider spelling out the strategy/intention based on your commit comment in the YAML. This makes it easier to see for others.

**Review by kasperg — APPROVED** [review]
:+1: I have some comments for you to consider before you merge this.

Then it will be interesting to see what happens 😅 .

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    minor-github-actions-dependencies:
+      update-types:
+        - "minor"
+        - "patch"
+  schedule:
+    interval: weekly
+    timezone: Europe/Copenhagen
+  labels:
+    - "dependencies"
+    - "github_actions"
```
[the remark this answers is not in the document]
Comment (spaceo) [reply .github/dependabot.yml]: I had another go on the configuration based on your comments. I think we have a better alternative now. I'll set the monster free and we'll se how it goes...

**Review by xendk — APPROVED** [review]

**Pull request merged**